nurseconnect/forms.py

```diff
@@ -24,7 +25,26 @@ class RegistrationForm(forms.Form):
                 "for": "mobilenum"
             }
         ),
-        label=_("Mobile Number")
+        label=_("Mobile Number"),
+        error_messages={
+            "invalid": "Please enter a valid South African cellphone number."
+        },
+    )
+
+    clinic_code = forms.RegexField(
```

Comment:
No need for those two. The regex pattern enforces it anyway. Having them would result in tow errors reported for the same thing.